### PR TITLE
makes uptimes and runtimes real

### PR DESCRIPTION
**Pull request by autodidaddict**

### CODE REVIEW

**Comment by autodidaddict** [comment]
Closes #61

File: internal/node/workload_mgr.go [hunk 1]
```diff
@@ -293,15 +293,39 @@ func (w *WorkloadManager) RunningWorkloads() ([]controlapi.MachineSummary, error
 	summaries := make([]controlapi.MachineSummary, len(procs))
 
 	for i, p := range procs {
+		uptimeFriendly := "unknown"
```
Comment (kthomas) [inline internal/node/workload_mgr.go:296]: Might be good to break this out into an `enrich()` method... someday.

File: internal/agent-api/client.go [hunk 2]
```diff
@@ -162,6 +166,19 @@ func (a *AgentClient) Undeploy() error {
 	return nil
 }
 
+func (a *AgentClient) RecordExecTime(elapsedNanos int64) {
+	a.execTotalNanos += elapsedNanos
```
Comment (kthomas) [inline internal/agent-api/client.go:170]: This isn't thread-safe. If a function is called at the same exact moment, there will be a race to set this.

Comment (kthomas) [reply internal/agent-api/client.go:170]: Could simply add an execTimeMutex in the agent and lock/defer unlock before `a.execTotalNanos += elapsedNanos`

Comment (autodidaddict) [reply internal/agent-api/client.go:170]: if this codebase gets one more mutex, I'm rewriting all the async code ;)

Comment (kthomas) [reply internal/agent-api/client.go]: Yeah, `atomic` is a better way to do it ;)

**Review by kthomas — APPROVED** [review]

File: internal/node/workload_mgr.go (hunk 1 above)
Comment (autodidaddict) [reply internal/node/workload_mgr.go:296]: there's already `myUptime`, but since I copied that from NATS server I didn't want to mess with it... yet.

**Pull request merged**